src/main/java/bio/terra/cli/businessobject/Resource.java

```diff
@@ -273,7 +273,8 @@ public enum Type {
     BQ_TABLE,
     AI_NOTEBOOK,
     GIT_REPO,
-    S3_STORAGE_FOLDER
+    S3_STORAGE_FOLDER,
```

Comment:
Convention seems to indicate it should be `SAGEMAKER_NOTEBOOK` since the cloud provider isn't mentioned in any of the others

Reply:
N/A. S3 was changed to AWS_S3 in a previous PR. This will reflect here after rebase